src/features/Dashboard/components/NoTransactions.tsx

```diff
@@ -55,17 +50,22 @@ const PlusIcon = styled.img`
 function NoTransactions() {
   return (
     <NoTransactionsWrapper>
-      <Link to="/send">
-        <NoTransactionsCenter>
-          <PlusIcon src={addIcon} />
-          <NoTransactionsHeading>
-            {translate('RECENT_TX_LIST_NO_TRANSACTIONS')}
-          </NoTransactionsHeading>
-          <NoTransactionsDescription>
-            {translate('RECENT_TX_LIST_NO_TRANSACTIONS_MORE')}
-          </NoTransactionsDescription>
-        </NoTransactionsCenter>
-      </Link>
+      <NoTransactionsCenter>
```

Comment:
Updated so that:
- text is the only link.
- cursor is limited to clickable zone. 
- copy use active voice.
<img width="920" alt="Screenshot 2021-02-10 at 17 20 14" src="https://user-images.githubusercontent.com/2429708/107538434-4593ac80-6bc4-11eb-99c2-9acf8f344617.png">